packages/ui-components/style/icons/jupyter/jupyter-favicon.svg

```diff
@@ -1,5 +1,5 @@
 <svg width="152" height="165" viewBox="0 0 152 165" version="1.1" xmlns="http://www.w3.org/2000/svg">
-  <g class="jp-icon-warn0" fill="#F37726">
+   <g class="jp-icon-selectable" fill="#F37726">
```

Comment:
```suggestion
   <g class="jp-icon-selectable jp-jupyter-icon-color" fill="#F37726">
```

Was the change to selectable intended here? Was omission of `jp-jupyter-icon-color` intended? I am not saying it is wrong, just asking for rationale. Otherwise it looks good to me.

Reply:
Thanks for the comment. Both changes were intended. In the last commit I removed the `jp-icon selectable` to set back to the initial case (to be honnest I am not sure to understand what it adds but I had bugs with having only fill and the color value so I had a look at patterns working for other icons) and `jp-jupyter-icon-color` was only introduced in commit [6c470c8](https://github.com/jupyterlab/jupyterlab/pull/13279/commits/6c470c808e13385fce5ff54d0e9d6c161f48e195) in which the ccs variable has been added in the `variables.cs`. I hope it answers your question.

Reply:
Thanks for a quick answer. I still see the `jp-icon selectable` in the last commit. Did you push the last commit? Just to make sure we are on the same page, I see `jp-jupyter-icon-color` added to `jupyter.svg`, but not to `jupyter-favicon.svg`.

Reply:
Sorry. After checking it appears that I forgot to save my local changes for `jupyter-favicon.svg`so the pushed file was not updated even if it looked ok on my computer. I think that now it should be ok.